src/Aion/PrivateKey.h

```diff
@@ -0,0 +1,65 @@
+// Copyright © 2017-2019 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include <array>
+#include <vector>
+
+namespace TW {
+namespace Aion {
+
+class PrivateKey {
```

Comment:
Why do we need a new implementation of `PrivateKey`?

Reply:
Original `PrivateKey` has different size and signs with secp256k1.

This solution is temporary, I haven't finished working on transaction signing and code structure yet.

I think I'll patch original `PrivateKey` to support Aion size.